src/openforms/contrib/kvk/tests/files/vcr_cassettes/KvKRemoteValidatorTestCase/KvKRemoteValidatorTestCase.test_branchNumber_validator_caching.yaml

```diff
@@ -0,0 +1,220 @@
+interactions:
+- request:
+    body: null
+    headers:
+      Accept:
+      - application/hal+json
+      Accept-Encoding:
+      - gzip, deflate, br
+      Connection:
+      - keep-alive
+      Content-Type:
+      - application/hal+json
+      User-Agent:
+      - python-requests/2.31.0
+      apikey:
+      - l7xx1f2691f2520d487b902f4e0b57a0b197
+    method: GET
+    uri: https://api.kvk.nl/test/api/v2/zoeken?vestigingsnummer=990000541921
+  response:
+    body:
+      string: '{"pagina":1,"resultatenPerPagina":10,"totaal":1,"resultaten":[{"kvkNummer":"90001354","vestigingsnummer":"990000541921","naam":"Grand
+        Joyflex B.V.","adres":{"binnenlandsAdres":{"type":"postadres","plaats":"Sterksel"}},"type":"hoofdvestiging","_links":{"basisprofiel":{"href":"https://api.kvk.nl/test/api/v1/basisprofielen/90001354"},"vestigingsprofiel":{"href":"https://api.kvk.nl/test/api/v1/vestigingsprofielen/990000541921"}}}],"_links":{"self":{"href":"https://api.kvk.nl/test/api/v2/zoeken?vestigingsnummer=990000541921&pagina=1&resultatenperpagina=10"}}}'
+    headers:
+      API-Version:
+      - 2.0.0
+      Cache-Control:
+      - no-cache, no-store, max-age=0, must-revalidate
+      Connection:
+      - keep-alive
+      Content-Encoding:
+      - gzip
+      Content-Length:
+      - '286'
+      Content-Security-Policy:
+      - frame-ancestors 'none', default-src 'none'
+      Content-Type:
+      - application/hal+json
+      Date:
+      - Thu, 02 May 2024 13:01:28 GMT
+      Feature-Policy:
+      - '''none'''
+      Keep-Alive:
+      - timeout=60
+      Referrer-Policy:
+      - no-referrer
+      Server:
+      - unknown
+      X-Content-Type-Options:
+      - nosniff
+      X-Frame-Options:
+      - DENY
+      strict-transport-security:
+      - max-age=157680000 ; includeSubDomains
+    status:
+      code: 200
+      message: ''
+- request:
```

Comment:
this request seems to be duplicated, it's the same as the one before, I think it's because of a particular VCR record mode that just records them again even though they're already present.

Probably best to just delete the cassettes and run the tests again :)

Reply:
Ah okay, I didn't know you had to delete them if you record new cassettes :+1: